Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-import AsyncHTTPClient
+@testable import AsyncHTTPClient
```

Comment:
i think it's better to move this test to `HTTPClientInternalTests`, this test class is intended to test public API methods

Reply:
https://github.com/swift-server/async-http-client/pull/67/commits/8775fc10569294c853f28403f23196226ad2ee08